### DISCUSSION TITLE
Polling allegedly flushes validation bag outside of component scope

### DISCUSSION THREAD

**Discussion by andriihorpenko** [livewire/livewire#4711] [7. Bugs (archived)]
Hey 👋 For this particular case I am using Livewire to continuously poll remaining time for another verification code request. This component is located deep inside a regular Blade view, so that it appears right inside a `<form>` block.

For some reason, when I add polling functionality to the component's root `<div>`, the validation bag gets flushed, so no form errors are displayed. However, when I remove polling, validation bag remains. This bugs me, as the component has nothing to do with the outer scope, it is solely responsible for polling remaining time from the backend and displaying it.

I might be missing something, but as for now I am puzzled. Let me know if I need to provide anything else 😊

```blade
{{-- 
When polling is added, error is not shown and "$errors->any()" returns false.
When polling is removed, everything is vice versa 
--}}

<form action="{{ route('verification.complete') }}" method="POST">
    @csrf
       
    <input type="text" name="code" />
    @error('code')
        <label>{{ $message }}</label>
    @enderror
    
    @livewire('resend-verification-code')

    <button>Confirm</button>
</form>
```

```blade
{{-- Livewire component view --}}

<div wire:poll.500ms="updateRemaining"> {{-- when "wire:poll..." is removed, validation works as expected --}}
    @if($remaining > 0)
        <span>{{ $remaining }}s</span>
    @endif
    <div wire:click="send">
        <svg>...</svg>
    </div>
</div>
```

```php
// Livewire component

class ResendVerificationCode extends Component
{
    public User $user;
    
    public int $remaining;

    public function mount()
    {
        $this->user = auth()->user();
        $this->updateRemaining();
    }

    public function updateRemaining()
    {
        $this->remaining = now()->diffInSeconds(cache($this->user->cooldownCacheKey()));
    }

    public function send()
    {
        if ($this->remaining > 0) {
            return;
        }

        $this->user->sendPhoneVerificationNotification();
    }

    public function render()
    {
        return view('livewire.resend-verification-code');
    }
}
```

**Comment by joshhanley** [comment]
@andriihorpenko ah interesting! Yeah Livewire definitely shouldn't touch the outer scope at all, only re-render what is inside the component. Any chance you could setup a demo on https://laravelplayground.com/#/ so we can have a look?

Thanks!
